db/models.py

```diff
@@ -14,3 +14,39 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="performers")
+    genres = models.ManyToManyField(Genre, related_name="category")
+
+    def __str__(self, description: str = None) -> str:
+        return f"{self.title}"
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self) -> str:
+        return f"{self.name}"
+
+    @property
+    def capacity(self) -> int:
+        return self.rows * self.seats_in_row
+
+
+class MovieSession(models.Model):
+    show_time = models.DateTimeField()
+    cinema_hall = models.ForeignKey(CinemaHall,
+                                    on_delete=models.DO_NOTHING,
+                                    null=True)
+    movie = models.ForeignKey(Movie,
+                              on_delete=models.DO_NOTHING,
```

Comment:
why do u use do nothing here, co if we'll delete a movie/cinemahall, movie session still can exist but without movie and cinema hall?